ethereum/listener.go

```diff
@@ -310,31 +314,32 @@ func (e *Ethereum) TrackLatestBlockHeight(ctx context.Context, logger log.Logger
 
 	d := fmt.Sprint(e.domain)
 
-	headers := make(chan *ethtypes.Header)
-
-	sub, err := e.wsClient.SubscribeNewHead(ctx, headers)
```

Comment:
I do want to note that DanK had issues with one of the ETH L2s which is why subscriptions were used iirc. Maybe less queries will help it. So if ARB or something has issues, this is likely the culprit

Reply:
Good to know, I'll put it through some extensive testing in testnet to make sure the flushing process still continues to work as expected without the subscriptions. Hopefully he was just having issues pulling in transactions with the subscriptions and not block height.